python-sdk/nuscenes/eval/panoptic/README.md

```diff
@@ -205,63 +207,71 @@ We also use PQ† [Porzi et al.](https://arxiv.org/pdf/1905.01220.pdf), which ma
+segments with track ID discrepancy, penalizing the incorrect track predictions. PTQ is defined as
 (∑{**1**(p,g) IoU(p,g)} - |IDS|) / (|TP|+ 0.5|FP|+ 0.5|FN|), where IDS stands for ID switches, and it is computed as
 the number of true positives (TP) that differ between tracking prediction and ground truth.
 
-#### Soft Panoptic Tracking Quality (sPTQ)
-Based on the same paper as PTQ, we also use a less strict version that penalizes the track ID discrepancy by
-subtracting the IoU scores at frames with ID switches instead of the total number of ID switches. The sPTQ metric is
-defined as: (∑{**1**(p,g) IoU(p,g)} - ∑(s)∈ IDS {s}) / (|TP|+ 0.5|FP|+ 0.5|FN|).
+From the same paper as PTQ, sPTQ (soft PTQ) penalizes the track ID discrepancy by subtracting the IoU scores at frames
+with ID switches instead of the total number of ID switches. The sPTQ metric is defined as:
+(∑{**1**(p,g) IoU(p,g)} - ∑(s)∈ IDS {s}) / (|TP|+ 0.5|FP|+ 0.5|FN|).
 
 Each score is calculated separately for each class, and then the mean is computed across classes. Note that points of
 class index 0 is ignored in the calculation.
 
-### LiDAR Segmentation and Tracking Quality (LSTQ)
-We also use LSTQ metric [Mehmet et al](https://arxiv.org/pdf/2102.12472.pdf). The LSTQ metric is computed as a geometric
-mean of the classification score and association score.
-
 ## Leaderboard
 nuScenes will maintain a single panoptic leaderboard with filters to split 4 specific tracks: Segmentation-lidar,
 Segmentation-open, Tracking-lidar and Tracking-open. Submissions of the first two panoptic segmentation tracks will be
-evaluated with segmentation metrics, while the two tracking submissions will be evaluated with both multi-object
-panopic tracking metrics as well as frame based panoptic segmentation metrics. For each submission the leaderboard will
+evaluated with segmentation metrics, while the two tracking submissions will be evaluated with both panoptic tracking
+metrics as well as frame based panoptic segmentation metrics. For each submission the leaderboard will
 list method aspects and evaluation metrics. Method aspects include input modalities (lidar, radar, vision), use of map
 data and use of external data. To enable a fair comparison between methods, the user will be able to filter the methods
 by method aspects.
 
-Both panoptic segmentation and multi-object panoptic tracking tasks will have an `Lidar track` and `Open Track`
-respectively. Methods will be compared within these tracks and the winners will be decided for each track separately.
-Furthermore, there will also be an award for novel ideas, as well as the best student submission.
+Both panoptic segmentation and panoptic tracking tasks will have an `Lidar track` and `Open Track` respectively.
+Methods will be compared within these tracks and the winners will be decided for each track separately. Furthermore,
+there will also be an award for novel ideas, as well as the best student submission.
```

Comment:
Let's remove `Furthermore, there will also be an award for novel ideas, as well as the best student submission.`